README.md

```diff
@@ -54,31 +94,43 @@ This module provisions a dataset and a table with an associated JSON schema.
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 
 ## Requirements
-### Terraform plugins
-- [Terraform](https://www.terraform.io/downloads.html) 0.12.x
-- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v2.5.0
-
-### Permissions
-In order to execute this module you must have a Service Account with the following roles:
- - roles/bigquery.dataOwner
-
-#### Script Helper
-A helper script for configuring a Service Account is located at (./helpers/setup-sa.sh).
```

Comment:
Let's also ckeep the reference to the helper script.

Reply:
Yes, makes sense, updated